### PR TITLE
Issue #769 Remove duplicated auto complete support in IDEA

### PR DESCRIPTION
**Pull request by dlsrb6342**
I tested this in local. There is no duplicated auto complete support.
image

### CODE REVIEW

File: resilience4j-spring-boot-common/build.gradle [hunk 1]
```diff
@@ -3,8 +3,8 @@ dependencies {
 
     compileOnly(libraries.spring_boot2_aop)
     compileOnly(libraries.spring_boot2_actuator)
-    annotationProcessor(libraries.spring_boot2_config_processor)
-    annotationProcessor(libraries.spring_boot2_autoconfigure_processor)
+    compileOnly(libraries.spring_boot2_config_processor)
```
Comment (RobWin) [inline resilience4j-spring-boot-common/build.gradle]: Why does that change behavior? Do you know it?

Comment (dlsrb6342) [reply resilience4j-spring-boot-common/build.gradle]: If we add `spring-boot-autoconfigure-processor` as annotationProcessor, it makes `spring-autoconfigure-metadata.json` and `spring-configuration-metadata.json`.
image

The one produced by the spring-boot-common module and the one created by the downstream module(like spring-boot, spring-boot2) are duplicated. So I changed spring-boot-common module not to make those files.

Comment (RobWin) [reply resilience4j-spring-boot-common/build.gradle]: But can't we then delete the dependencies?

Comment (dlsrb6342) [reply resilience4j-spring-boot-common/build.gradle]: Yes you are right. I removed them

**Pull request merged**